zscaler/icon_zscaler/actions/blacklist_url/action.py

```diff
@@ -24,6 +24,7 @@ def run(self, params={}):
         normalized_urls = []
         for url in urls:
             if url and not url.startswith("http"):
+                self.logger.info("URL did not start with http prefix, prefixing URL...")
```

Comment:
We should log the prefix too. Did not test:

```suggestion
                self.logger.info("fURL did not begin with protocol, prefixing with http://{url} ...")
```